pkg/services/featuremgmt/registry.go

```diff
@@ -707,9 +707,10 @@ var (
 		{
 			Name:         "dashgpt",
 			Description:  "Enable AI powered features in dashboards",
-			Stage:        FeatureStageExperimental,
+			Stage:        FeatureStageGeneralAvailability,
```

Comment:
Investigated making this public preview status instead and it turns out that in order for a feature flag to be enabled by default it must be GA status. In this case I think this is ok as features are stable and is behind LLM plugin install, configuration, and enablement.